### ISSUE TITLE
Javascript error when viewing a repo

### ISSUE THREAD

**Issue by rubenelshof** [go-gitea/gitea#32797]
### Description

Upon viewing a repository I get the following Javascript error:

`JavaScript error: Cannot read properties of null (reading 'addEventListener') (https://git.example.com/assets/js/index.js?v=1.23.0~dev-792-g39a01016cd @ 111:29734). Open browser console to see more details.`

Console error:
```
repo-common.ts:79 Uncaught TypeError: Cannot read properties of null (reading 'addEventListener')
    at uh (repo-common.ts:79:10)
    at Tv (repo-common.ts:95:3)
    at He (dom.ts:65:7)
    at ue (dom.ts:91:10)
    at Lv (repo-common.ts:106:3)
    at Ty (repo-legacy.ts:82:3)
    at y1 (index.ts:113:7)
    at HTMLDocument.<anonymous> (index.ts:123:3)
```

### Gitea Version

1.23.0+dev-792-g39a01016cd

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots

![gitea error](https://github.com/user-attachments/assets/69577c3f-080a-49bf-9e6b-37cb1429d34d)

### How are you running Gitea?

Docker

### Database

None

**Comment by lunny** [comment]
Maybe caused by #31142

**Comment by rubenelshof** [comment]
> Maybe caused by #31142

It could be. 
When I click the 'Code' button it doesn't give me the dropdown menu as shown in the screenshots of that PR.

**Comment by wxiaoguang** [comment]
Will fix it

**Comment by wxiaoguang** [comment]
->  Fix clone panel js error #32798
